files/default/neutron-ha-tool.py

```diff
@@ -579,6 +579,16 @@ def list_agents(qclient, agent_type=None):
     return resp['agents']
 
 
+def list_alive_agents(qclient, agent_type=None):
+    resp = qclient.list_agents()
+    LOG.debug("list_agents: %s", resp)
+    if agent_type:
+        return [agent for agent in resp['agents']
+                if agent['agent_type'] == agent_type and
+                agent['alive'] is True]
```

Comment:
Oh right, you confused me by commenting on line 588 not line 585 :-)  I can't imagine why `agent_type` would ever be `None`, and indeed in the current code it will only ever be `'L3 agent'`.  So this `if agent_type` seems redundant to me.